fakeserver/server.go

```diff
@@ -287,3 +345,33 @@ func generateSelfSignedCert() (tls.Certificate, error) {
 
 	return cert, nil
 }
+
+// Stub implementation for GetConfigurationUpdates
+func (s *server) GetConfigurationUpdates(stream pb.KubernetesInfoService_GetConfigurationUpdatesServer) error {
+	logger.Info("GetConfigurationUpdates stream started")
+	for {
+		// Simulate receiving updates (placeholder logic)
+		select {
+		case <-stream.Context().Done():
+			logger.Error("GetConfigurationUpdates stream context canceled", zap.Error(stream.Context().Err()))
+			return stream.Context().Err()
+		case <-time.After(10 * time.Second):
+			logger.Info("Sending keepalive on GetConfigurationUpdates stream")
+			// Placeholder: Send a keepalive or response
+		}
+	}
+}
+
+func (s *server) SendKubernetesNetworkFlows(stream pb.KubernetesInfoService_SendKubernetesNetworkFlowsServer) error {
+	logger.Info("SendKubernetesNetworkFlows stream started")
+	for {
```

Comment:
You have to read the messages received from the stream.
Right now, it's both network flows and keepalives.
https://github.com/illumio/cloud-operator/blob/df44b593931b44dd34b00bfa7b25eb248c9f90a5/api/illumio/cloud/k8sclustersync/v1/k8s_info.proto#L362-L369